agent/config/config.go

```diff
@@ -355,6 +357,40 @@ func fileConfig() (Config, error) {
 	return cfg, nil
 }
 
+// userDataConfig reads configuration JSON from instance's userdata. It doesn't
+// return any error as it's entirely optional to configure the ECS agent using
+// this method.
+// Example:
+// {"ECSAgentConfiguration":{"Cluster":"default"}}
+func userDataConfig(ec2Client ec2.EC2MetadataClient) Config {
+	type userDataParser struct {
+		Config Config `json:"ECSAgentConfiguration"`
+	}
+
+	parsedUserData := userDataParser{
+		Config: Config{},
+	}
+
+	userData, err := ec2Client.GetUserData()
+	if err != nil {
+		seelog.Warnf("Unable to fetch user data: %v", err)
+		// Unable to read userdata from instance metadata. Just
+		// return early
```

Comment:
Does "return early" have to be in its own comment line? Multiple occurences in this file.